    window.resources = [
    {
        field: 'Computational genomics',
        sources: [
            {
                name: 'Hu, Taishan, et al. "Next-generation sequencing technologies: An overview." Human Immunology 82.11 (2021): 801-811.',
                link: 'https://www.sciencedirect.com/science/article/pii/S0198885921000628?casa_token=Q069J2zdT2wAAAAA:8CtYPptaE3GWbEJQDqUOP-v_f-ce4-trsZZqC2d8hvF3LtLBOKQMJvENDG31Ksw83wA9Njf_bA',
                type: 'journal article',
                description: 'Overview of the main platforms for sequencing, i.e. platforms which produce data which are then analyzed withing computational genomics pipelines.',
            },
            {
                name: 'Sohn, Jang-il, and Jin-Wu Nam. "The present and future of de novo whole-genome assembly." Briefings in bioinformatics 19.1 (2018): 23-40.',
                link: 'https://academic.oup.com/bib/article/19/1/23/2339783?login=false',
                type: 'journal article',
                description: 'A review of genome assembly approaches. The review describes main idea of the alrogithms that are used in the field as well as the paradigm shift caused by the emergence of long-read technologies.',
            }, {
                name: 'Giani, Alice Maria, et al. "Long walk to genomics: History and current approaches to genome sequencing and assembly." Computational and Structural Biotechnology Journal 18 (2020): 9-19.',
                link: 'https://www.sciencedirect.com/science/article/pii/S2001037019303277',
                type: 'journal article',
                description: 'A more high-level review of genome assembly approaches putting them in the context of the evoluation of sequencing platforms and sequencing projects.',
            }, {
                name: 'Olson, Nathan D., et al. "Variant calling and benchmarking in an era of complete human genome sequences." Nature Reviews Genetics (2023): 1-20.',
                link: 'https://www.nature.com/articles/s41576-023-00590-0',
                type: 'journal article',
                description: 'A paper describing overviewing new developments in the field of variant calling, i.e. exploring sequence differentces in the genome of an individual with respect to a reference genome assembly.',
            }, {
                name: 'Wong, Aaron K., et al. "Decoding disease: from genomes to networks to phenotypes." Nature Reviews Genetics 22.12 (2021): 774-790.',
                link: 'https://www.nature.com/articles/s41576-021-00389-x',
                type: 'journal article',
                description: 'An easy-to-read high-level view of how (variation in) sequencing data can be linked to (changes in) phenotype. In this sense the article links genomics to systems biology (later parts of the paper). Specifically, it lists key challenges and opportunities in identifying specific disease-causing genetic variants and linking them to molecular pathways and, ultimately, to disease phenotypes',
            }, 
            {
                name: 'Escobar-Zepeda, Alejandra, Arturo Vera-Ponce de León, and Alejandro Sanchez-Flores. "The road to metagenomics: from microbiology to DNA sequencing technologies and bioinformatics." Frontiers in genetics 6 (2015): 348.',
                link: 'https://www.frontiersin.org/articles/10.3389/fgene.2015.00348/full',
                type: 'journal article',
                description: "Metagenomics is the study of genetic material collected from environmental or biological samples, enabling the analysis of entire microbial communities' genetic diversity and functional potential without the need for individual organism cultivation. This review highlights major milestones that led to modern metagenomics",
            },
            {
                name: 'Meyer, Fernando, et al. "Critical assessment of metagenome interpretation: the second round of challenges." Nature methods 19.4 (2022): 429-440.',
                link: 'https://www.nature.com/articles/s41592-022-01431-4',
                type: 'journal article',
                description: 'An article evaluating different metagenomic approaches within the Critical Assessment of Metagenome Interpretation (CAMI) challange, giving the overview of the state-of-the-art in the field.',
            },
            // {
            //     name: '',
            //     link: '',
            //     type: '',
            //     description: '',
            // },            

        ]
    }, {
        field: 'Structural bioinformatics',
        sources: [
            {
                name: 'de Almeida Paiva, Vinícius, et al. "Protein structural bioinformatics: An overview." Computers in Biology and Medicine 147 (2022): 105695.',
                link: 'https://www.sciencedirect.com/science/article/pii/S0010482522004784?casa_token=S9S1Qaj9i5UAAAAA:Yl1nrGWF0816Zse0qfrwvHp7rUa8rxiMs9S6RD8Lhcyv2rCTr0UrV7AC_s7_MZKjEE3hw7jD4A',
                type: "journal article",
                description: "General overview of the most common branches of structural bioinformatics, including docking, molecular dynamics, molecular visualization, structure prediction, mutation analysis, catalytic and binding site prediction, and databases.",
            },
            {
                name: 'Durairaj, Janani, Dick de Ridder, and Aalt DJ van Dijk. "Beyond sequence: Structure-based machine learning." Computational and Structural Biotechnology Journal (2022).',
                linka: 'https://www.sciencedirect.com/science/article/pii/S2001037022005967',
                type: 'journal article',
                description: "An overview of problems solved by machine learning appraoches in structural bioinformatics."
            },
            {
                name: 'Sali, Andrej. "From integrative structural biology to cell biology." Journal of Biological Chemistry 296 (2021).',
                linka: 'https://www.jbc.org/article/S0021-9258(21)00533-0/',
                type: 'journal article',
                description: "A review of integrative modeling approach to structural biology. It describes the goals, workflow, and challenges of integrative modeling."
            }

        ]
    }, {
        field: 'Computational proteomics',
        sources: [ 
            {
                name: "Found in translation: functions and evolution of a recently discovered alternative proteome",
                description: "A review of transcriptomics / translatomics",
                type: "journal article",
                link: "https://pubmed.ncbi.nlm.nih.gov/25795211/"
            }

        ]
    }, {
        field: 'Phylogenetics',
        sources: [
            {
                name: 'Yang, Ziheng, and Bruce Rannala. "Molecular phylogenetics: principles and practice." Nature reviews genetics 13.5 (2012): 303-314.',
                description: 'This review discusses the significant role of phylogenetic trees in modern biology. Phylogenies are used to understand gene family relationships, population histories, pathogen dynamics, cancer development, and more. The paper outlines the array of methods for phylogenetic inference and provides guidance for choosing the appropriate method for specific questions and data types, particularly in the era of next-generation sequencing.',
                type: "journal article",
                link: "https://www.nature.com/articles/nrg3186"
            },
            {
                name: 'Kapli, Paschalia, Ziheng Yang, and Maximilian J. Telford. "Phylogenetic tree building in the genomic age." Nature Reviews Genetics 21.7 (2020): 428-444.',
                description: 'Unlike the previous paper, which discuss more mathematical and computatinoal aspect of phylogenetics, this review provides more general background, discussing the major steps of phylogenetic analysis, including identification of orthologous genes or proteins, multiple sequence alignment, and choice of substitution models and inference methodologies',
                type: "journal article",
                link: "https://www.nature.com/articles/s41576-020-0233-0"
            },
            {
                name: 'De Bruyn, Alexandre, Darren P. Martin, and Pierre Lefeuvre. "Phylogenetic reconstruction methods: an overview." Molecular Plant Taxonomy: Methods and Protocols (2014): 257-277.',
                description: 'A simplified step-by-step guide to the construction of phylogenetic trees from nucleotide sequences using fairly up-to-date maximum likelihood methods implemented in freely available computer programs.',
                type: 'book chapter',
                link: 'https://link.springer.com/protocol/10.1007/978-1-62703-767-9_13'

            },
            // {
            //     name: 'Lartillot, Nicolas. "Probabilistic models of eukaryotic evolution: time for integration." Philosophical Transactions of the Royal Society B: Biological Sciences 370.1678 (2015): 20140338.',
            //     description: '',
            //     type: 'journal article',
            //     link: 'https://royalsocietypublishing.org/doi/full/10.1098/rstb.2014.0338'

            // },
            {
                name: 'Bromham, Lindell, et al. "Bayesian molecular dating: opening up the black box." Biological Reviews 93.2 (2018): 1165-1191.',
                description: 'An article overviewing the baysian-based molecular dating (estimation of the age of evolutionary events) methods. It describes the components of the dating methods, the important decisions that researchers must make in their analyses, and the factors that  need to be considered when interpreting results.',
                type: 'journal article',
                link: 'https://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12390?casa_token=azFIueE-hMIAAAAA:MsAnboWkL0IhkPzPoVRWBQgxzjo47jQCZSHgtWYuiFlLk9Rf1oRZrbTZ55TltecUbKzpmaagYd2ftQ'

            },
            
            

        ]
    }, {
        field: 'Population genetics',
        sources: [
            {
                name: 'Klein, Richard G. "Profile of Svante Pääbo: 2022 Nobel laureate in physiology or medicine." Proceedings of the National Academy of Sciences 120.1 (2023): e2217025119.',
                description: "The article delves into the reasons behind Svante Pääbo's Nobel Prize win in 2022. Svante Pääbo's work in human and Neanderthal population and evolutionary genetics offers an insight into the ongoing relevance and promising prospects of this field.",
                type: "journal article",
                link: "https://www.pnas.org/doi/10.1073/pnas.2217025119?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed"
            },
            {
                name: 'Casillas, Sònia, and Antonio Barbadilla. "Molecular population genetics." Genetics 205.3 (2017): 1003-1035.',
                description: 'This review describes the main concepts, methods, and landmarks of molecular population genetics, using the Drosophila model as a reference. It includes the description of the different genetic data sets made available by advances in molecular technologies, and the theoretical developments fostered by these data.',
                type: "journal article",
                link: "https://academic.oup.com/genetics/article/205/3/1003/6066925?login=false"
            },

        ]
    }, {
        field: 'Systems biology',
        sources: [
            {
                name: 'Kitano, Hiroaki. "Computational systems biology." Nature 420.6912 (2002): 206-210.',
                link: 'https://www.nature.com/articles/nature01254',
                type: 'A seminar paper, delineating the field of systems biology.',
                description: '',
            },
            {
                name: 'Bruggeman, Frank J., and Hans V. Westerhoff. "The nature of systems biology." TRENDS in Microbiology 15.1 (2007): 45-50.',
                link: 'https://www.cell.com/trends/microbiology/fulltext/S0966-842X(06)00264-2?large_figure=true',
                type: 'journal article',
                description: 'High-level explanation of the bottom-up and top-down approaches in systems biology and description of how they can be used to elucidate mechanisms and principles that underlie cell function',
            },
           

        ]
    }, {
        field: 'Neuroinformatics/Computational neuroscience',
        sources: [
            {
                name: "Scholarpedia: Computational neuroscience",
                link:'http://www.scholarpedia.org/article/Encyclopedia_of_computational_neuroscience',
                type: 'web resource',
                description: 'Comprehensive resource covering experimental and theoretical neuroscience as well as computational aspects of the field.'
            }

        ]
    }, {
        field: 'Bioimaging',
        sources: [
            {
                name: 'Moen, Erick, et al. "Deep learning for cellular image analysis." Nature methods 16.12 (2019): 1233-1246.',
                link: 'https://www.nature.com/articles/s41592-019-0403-1',
                type: 'journal article',
                description: 'A review article describing applications of deep learning and progress in four key applications: image classification, image segmentation, object tracking, and augmented microscopy.'
            }

        ]
    }, {
        field: 'Bioactive molecule discovery/Computational drug discovery',
        sources: [ 
            {
                name: 'Vamathevan, Jessica, et al. "Applications of machine learning in drug discovery and development." Nature reviews Drug discovery 18.6 (2019): 463-477.',
                link: 'https://www.nature.com/articles/s41573-019-0024-5',
                type: 'journal article',
                description: 'An extensively cited paper provides a comprehensive overview of all stages in the drug discovery pipeline, highlighting the successful applications of machine learning to various problems.'
            
            }

        ]
    }, {
        field: 'Molecular modeling',
        sources: [
            {
                name: 'Bunker, A., & Róg, T. (2020). Mechanistic understanding from molecular dynamics simulation in pharmaceutical research 1: Drug delivery. Frontiers in Molecular Biosciences, 7, 604770.',
                link: 'https://www.frontiersin.org/articles/10.3389/fmolb.2020.604770/full',
                type: 'journal article',
                description: 'First part of a review covering pharmaceutical and computational background of molecular modeling. The application focus is on nanomedicine, but the description of hte methods is very pedagogical and easy to digest.'
            },
            {
                name: 'Róg, Tomasz, Mykhailo Girych, and Alex Bunker. "Mechanistic understanding from molecular dynamics in pharmaceutical research 2: lipid membrane in drug design." Pharmaceuticals 14.10 (2021): 1062.',
                link: 'https://www.mdpi.com/1424-8247/14/10/1062',
                type: 'journal article',
                description: 'Second part of the review covering concepts such as metadynamics or umbrella sampling.'
            },
            {
                name: 'Kmiecik, Sebastian, et al. "Coarse-grained protein models and their applications." Chemical reviews 116.14 (2016): 7898-7936.',
                link: 'https://pubs.acs.org/doi/full/10.1021/acs.chemrev.6b00163',
                type: 'journal article',
                description: 'Many protein systems are too large to be studied in atomistic detail. This review provides an overview of coarse-grained models focusing on their design, including choices of representation, models of energy functions, sampling of conformational space, and applications in the modeling of protein structure, dynamics, and interactions.'
            }
            

        ]
    }, 
    
]