<!DOCTYPE html>
<html lang="en">
<head>
      <title>Binformatics seminar</title>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

      <!-- icons -->
      <link rel="shortcut icon" href="favicon.ico">
      <link rel="icon" href="favicon.ico">

      <!-- Override CSS file - add your own CSS rules -->
      <link rel="stylesheet" href="styles.css">

    </head>
    <body>
    <div class="content">
      <div class="container">
        <div class="main">
          <p><img class="img-responsive" src="michelangelo-grey.jpg" alt=""></p>


          <h1>Bioinformatics seminar</h1>
          <hr>
          <h3><b>Next lecture</b></h3>
          <div class="prednaska">
            <b>Victoria Offord (WSI)</b><br>
            <b><i>Functionalizing the cancer genome base-by-base with CRISPR editing and computational tools</i></b><br><br>
            <p>
              Over the last decade, the reduction in costs for high throughput technologies has enabled us to sequence more human genomes than ever before. We can see this in the rapid expansion of the repositories used to store and share genetic data amongst researchers. To date, over 1.5 million variants have been submitted to ClinVar, a public database of clinically relevant variants. However, over 50% of these are variants of uncertain significance (VUS) for which we have no clear interpretation of their function. Multiplexed Assays of Variant Effect (MAVEs) can be used to generate variant effect maps, which, over time, may be used to aid clinical interpretations. Saturation genome editing (SGE) is a CRISPR-based MAVE which allows us to test all possible single- and multi-nucleotide or amino acid variations across a genomic region. We’ll discuss SGE, the community, available resources, tools being developed and how the data that we’ve been generating may go on to aid in the interpretation of VUS.
            </p>
            Wednesday <b>7. 12. 2022</b> 17:20, lecture hall <b>S3</b>, Malostranské nám. 25

          </div>


          <h3><b>Program</b></h3>
          <div class="block">
            <table>              
              <tr><td class="date"> 7. 12. 2022    </td><td>Victoria Offord (WSI)</td><td>Functionalizing the cancer genome base-by-base with CRISPR editing and computational tools</td></tr>
            </table>
            <a href="https://calendar.google.com/calendar/embed?src=v12usoaq5hqtdu2nbkpt1fo1v4%40group.calendar.google.com&ctz=Europe%2FPrague"><b>Lectures calendar</b></a>
            <br>

            Sending information about next lecture via  <a href="https://kam.mff.cuni.cz/mailman3/postorius/lists/bioinf-l.kam.mff.cuni.cz/"><b>mailing list</b></a>
            <br>
            <a href="previous.html">Past lectures</a>
          </div>


          <hr>
          <h3><b>Contact</b></h3>
          <div class="block">
            Managed by departments of
            <ul>
              <li><a href="https://www.ksi.mff.cuni.cz/">Software Engineering</a>, Faculty of Mathematics and Physics, Charles University (Malostranské nám. 25, 118 00 Praha 1 - Malá Strana)</li>
              <li><a href="https://www.natur.cuni.cz/biologie/bunecna-biologie">Cell Biology</a>, Faculty of Science, Charles University (Viničná 7, 128 00 Praha 2)</li>
            </ul>
            Course code: MB151S01

            <br>
            <p>
              <br> <a href="https://www.sanger.ac.uk/person/danecek-petr/">Petr Daněček</a>
              <br> <a href="https://bioinformatics.cuni.cz/hoksza">David Hoksza</a>
              <br> <a href="https://www.natur.cuni.cz/biologie/bunecna-biologie/osobni-stranky/marian-novotny">Marian Novotný</a>
            </p>
          </div>

        </div>
      </div>
    </div>
    </body>
    </html>

