<!DOCTYPE html>
<html lang="en">
<head>
      <title>Binformatics seminar</title>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

      <!-- icons -->
      <link rel="shortcut icon" href="favicon.ico">
      <link rel="icon" href="favicon.ico">

      <!-- Override CSS file - add your own CSS rules -->
      <link rel="stylesheet" href="styles.css">

    </head>
    <body>
    <div class="content">
      <div class="container">
        <div class="main">
          <p><img class="img-responsive" src="michelangelo-grey.jpg" alt=""></p>


          <h1>Bioinformatics seminar</h1>
          <hr>
          <h3><b>Next meeting</b></h3>
          <div class="prednaska">
            <div class="title">              
              How does the structure of a genotype-phenotype map shape variation and evolutionary processes?              
            </div>
            <div class="abstract">                             
              Evolutionary processes are relevant in many areas of the life sciences, and since these processes are fundamentally quantitative and stochastic, mathematical and computational models are required to capture their dynamics. Such models need two central elements: variation through random mutations and natural selection. To connect variation and selection in a single model, we need to translate random mutations on the level of genotypes to their selectively relevant phenotypic consequences. This connection is given by genotype-phenotype (GP) maps, datasets translating any possible genotype to its phenotype. GP maps are large (with 4^L genotypes of length L) and thus well-studied GP map examples are typically computational models of molecular phenotypes (for example, RNA secondary structures). However, further computational and empirical GP maps for phenotypes on the molecular scale and beyond are also becoming available. Therefore, it is important to take a systematic approach that can be applied to many GP maps: This approach should identify coarse-grained features that characterise a GP map, and investigate, how they shape evolutionary processes on this map. In this talk, I will illustrate this with two examples [1, 2]: (1) How does one specific feature called “genetic correlations”, which had been reported in several GP maps, shape evolutionary processes? (2) What are the topographies of GP-map-based fitness landscapes (in the baseline case of random phenotype-fitness assignments); can we break these down to specific GP map features? 
 
              <ol>
                <li>N. S. Martin, S. Schaper, C.Q. Camargo, A. A. Louis. Mol. Biol. Evol. 41(6):msae085 (2024).</li>
                <li>M. Srivastava, A.A. Louis, N.S. Martin. bioRxiv (2025).</li>
              </ol>
            </div>            
            <div>
              <a href="https://www.crg.eu/en/content/research/barcelona-collaboratorium-independent-fellow/nora-martin">Nora Martin</a> (Centre for Genomic Regulation)<br><br>
              <!-- Patrícia Sotáková, Miloš Halda<br><br> -->
              <!-- Tuesday <b>4. 3. 2025</b> 14:50, lecture room <b>Fotochemie</b>, Viničná 7 -->
              Wednesday <b>30. 4. 2025</b> 17:20, lecture room <b>S4</b>, Malostranské nám. 25
            </div>
          </div>
          <h3><b>Program</b></h3>
          <div class="block">
            <table>                                             
               <tr><td class="date">7. 5. 2025</td><td>Master students' presentations (Sára Štráchalová, Lucie Drahoňovská)</td><td></td></tr>
               <tr><td class="date">14. 5. 2025</td><td>TBA</td><td></td></tr>
            </table>
            <a href="https://calendar.google.com/calendar/embed?src=v12usoaq5hqtdu2nbkpt1fo1v4%40group.calendar.google.com&ctz=Europe%2FPrague"><b>Invited guest lectures calendar</b></a>
            <br>

            Sending information about next lecture via  <a href="https://kam.mff.cuni.cz/mailman3/postorius/lists/bioinf-l.kam.mff.cuni.cz/"><b>mailing list</b></a>
            <br>
            <a href="past.html">Past lectures</a>
          </div>


          <hr>
          <h3><b>Doctoral bioinformatics seminar information</b></h3>
          <div class="block">
            The seminar is running concurrently with the Doctoral bioinformatics seminar (<a href="https://is.cuni.cz/studium/eng/predmety/index.php?do=predmet&kod=NSWI201">NSWI201</a>) specifically designed for PhD students in the Bioinformatics and Computational Biology doctoral study program. More information for PhD students is available on a <a href="phd.html">separate website</a>.
          </div>
          <hr>
          <h3><b>Contact</b></h3>
          <div class="block">
            Managed by departments of
            <ul>
              <li><a href="https://www.ksi.mff.cuni.cz/">Software Engineering</a>, Faculty of Mathematics and Physics, Charles University (Malostranské nám. 25, 118 00 Praha 1 - Malá Strana)</li>
              <li><a href="https://www.natur.cuni.cz/biologie/bunecna-biologie">Cell Biology</a>, Faculty of Science, Charles University (Viničná 7, 128 00 Praha 2)</li>
            </ul>
            Course codes: MB151S01, MB151S02, MB151S03, MB151S04, NSWI201

            <br>
            <p>
              <br> <a href="https://www.sanger.ac.uk/person/danecek-petr/">Petr Daněček</a>
              <br> <a href="https://bioinformatics.cuni.cz/hoksza">David Hoksza</a>
              <br> <a href="https://natur.cuni.cz/osoba?poid=1842408462118459">Marian Novotný</a>
            </p>
          </div>

        </div>
      </div>
    </div>
    </body>
    </html>

