
<!DOCTYPE html>
<html  lang="en">
<head>
    <title>Bioinformatics seminar</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

	<!-- icons -->
	<link rel="shortcut icon" href="favicon.ico">
	<link rel="icon" href="favicon.ico">

	<!-- Override CSS file - add your own CSS rules -->
	<link rel="stylesheet" href="styles.css">

</head>

<body>
<div class="content">
<div class="container">
<div class="main">
    <p><a href="index.html"><img class="img-responsive" src="michelangelo-grey.jpg" alt=""></a></p>


    <h3><b>Past invited lectures</b></h3>
    <hr>
    <table>
        <tr><td colspan="3"><b>2024/2025</b></td></tr>
        <tr><td class="date">26. 2. 2025</td><td>Daniel Žucha (IBT CAS)</td><td>Spatial Transcriptomics In Health and Disease: A Stroke Case Study</td></tr>
        <tr><td class="date">17. 1. 2025</td><td>Gerardo Tauriello (SIB)</td><td>Beyond SWISS-MODEL - Modelling, Benchmarking and Working with Computed Structure Models in the AlphaFold Era</td></tr>
        <tr><td class="date">27. 11. 2024</td><td>Karel Sedlář (VUT)</td><td>Computational Analyses and Functional Annotations of Non-Model Bacteria</td></tr>
        <tr><td class="date">27. 11. 2024</td><td>Anu Shivalikanjli (EMBL-EBI)</td><td>Gene Off, Insights On: Mapping Molecular Phenotypes with MorPhiC</td></tr>
        <tr><td class="date">6. 11. 2024</td><td>Hana Rozhoňová (Charles University)</td><td>The genetic code and protein evolvability</td></tr>
        <tr><td colspan="3"><b>2023/2024</b></td></tr>
        <tr><td class="date">14. 5. 2024</td><td>Tom Williams (University of Bristol)</td><td>Phylogenomics and the early evolution of life</td></tr>
        <tr><td class="date">24. 4. 2024</td><td>Jan Řezáč (IOCB, Prague)</td><td>Semiempirical Quantum Mechanical Scoring in Structure-based Drug Design</td></tr>
        <tr><td class="date">17. 4. 2024</td><td>Geoff Barton (University of Dundee)</td><td>From human population variation to ligand binding sites via SARS-CoV-2</td></tr>
        <tr><td class="date">10. 4. 2024</td><td>Karolína Korvasová (MFF UK)</td><td>Computational methods for prosthetic vision: inferring functional structure of the brain from its spontaneous activity</td></tr>
        <tr><td class="date">3. 4. 2024</td><td>Lukáš Valihrach (IBT, BioCev)</td><td>Revolutionizing Modern Biomedical Research with Single-Cell and Spatial Transcriptomics</td></tr>
        <tr><td class="date">20. 3. 2024</td><td>Bailey Francis (Sanger)</td><td>A Telomere-to-Telomere (T2T) Mouse Genome Assembly: Towards the first complete mouse genome</td></tr>
        <tr><td class="date">5. 3. 2024</td><td>Hana Rozhoňová (ETH Zurich)</td><td>A computational screen for alternative genetic codes in viruses</td></tr>
        <tr><td class="date">20. 12. 2023</td><td>Karel Brinda (INRIA)</td><td>Efficient Search of Microbial Genomes via Phylogenetic Compression</td></tr>
        <tr><td class="date">12. 12. 2023</td><td>Rafael Najmanovich (University of Montreal)</td><td>Probing the dynamics of macromolecules and energetics of molecular interactions with high-performance, fast and accessible computational methods</td></tr>
        <tr><td class="date">22. 11. 2023</td><td>Jana Musilová (Brno University of Technology)</td><td>Augusta, a Python package for inferring Gene Regulatory and Boolean Networks using RNA-Seq and data mining</td></tr>
        <tr><td class="date">25. 10. 2023</td><td>Lenka Gahurová (University of South Bohemia)</td><td>(Un)usual characteristics of African mole-rats' oocytes and sex chromosomes through genomics and transcriptomics approaches</td></tr>
        <tr><td class="date">18. 10. 2023</td><td>Tomáš Pluskal, Ela Nekardová, Lukáš Valihrach, Jiří Lindovský, David Hoksza</td><td>Projects opportunities</td></tr>
        <tr><td class="date">11. 10. 2023</td><td>Michal Kolář, Jan Stuchlý, Jan Antolík, Martin Modrák, Marian Novotný</td><td>Projects opportunities</td></tr>

        <tr><td colspan="3"><b>2022/2023</b></td></tr>
        <tr><td class="date">10. 5. 2023</td><td>Carl Hermann (University of Heidelberg)</td><td>Machine learning approaches for the identification of disease signatures</td></tr>
        <tr><td class="date">2. 5. 2023</td><td>Erich Bornberg-Bauer (University of Muenster)</td><td>Lost (and found) in translation: How evolution can find functional proteins in vast sequence space</td></tr>
        <tr><td class="date">19. 4. 2023</td><td>Kateřina Storchmannova (Palacký University Olomouc)</td><td>MolMeDB - databáze interakcí malých molekul s membránami a membránovými transportními proteiny</td></tr>
        <tr><td>12. 4. 2023</td><td></td><td>Discussion - Ten simple rules for defining a computational biology project</td></tr>
        <tr><td>5. 4. 2023</td><td>Lenka Backová (UPV/EHU)</td><td>Bioimaging</td></tr>
        <tr><td>29. 3. 2023</td><td>Ondřej Sladký (Faculty of Mathematics and Physics, Charles University)</td><td>Masked superstrings as a unified framework for textual k-mer set representations</td></tr>
	    <tr><td>22. 3. 2023</td><td><a href="https://wwwen.uni.lu/lcsb/people/marek_ostaszewski">Marek Ostaszewski</a> (Luxembourg Centre for Systems Biomedicine)</td><td>Disease maps: building and analysing graphical models of biomedical knowledge</td></tr>      
        <tr><td>8. 3. 2023</td><td>Joern Gerchen (Faculty of Science, Charles University)</td><td>Building complex bioinformatics pipelines using Snakemake</td></tr>
              
	    <tr><td>7. 12. 2022</td><td>Victoria Offord (WSI)</td><td>Functionalizing the cancer genome base-by-base with CRISPR editing and computational tools</td></tr>
        <tr><td>23. 11. 2022</td><td><a href="https://www.ebi.ac.uk/people/person/mihaly-varadi/">Mihaly Varadi</a> (EMBL-EBI)</td><td>AlphaFold DB: Increasing the structure coverage of
            the sequence space by a thousand times</td></tr>
        <tr><td class="date">19. 10. 2022<td><a href="http://neuro.felk.cvut.cz/team/#tomas-sieger">Tomáš Sieger</a> (compNeuroGroup, ČVUT)<td>Measures of quality of clusters in hierarchical clustering of flow cytometry data</tr>
        <tr><td colspan="3"><b>2021/2022</b></td></tr>
        <tr><td>19. 10. 2022</td><td>Tomáš Sieger</td><td>Measures of quality of clusters in hierarchical clustering of flow cytometry data</td></tr>
        <tr><td> 4. 5. 2022 </td><td>Alessandra Carbone</td><td> Sequence space and protein functional classification</td></tr>
        <tr><td> 30. 3. 2022 </td><td>Giulio Vistoli</td><td> Novel strategies for virtual screening and drug repurposing: the lesson learned from the SARS-CoV-2 pandemy</td></tr>
        <tr><td> 23. 2. 2022 </td><td>Eugene Gardner</td><td> Investigating the Genetics of Sexual Reproduction in 200,000 Humans</td></tr>
        <tr><td> 24. 11. 2021 </td><td> Abdullah Kahraman</td><td> MTPpilot: an interactive software for NGS results analysis for molecular tumor boards</td></tr>
        <tr><td> 12. 10. 2021 </td><td> Augustin Žídek</td><td> Deepmind: Protein Structure Prediction with AlphaFold</td></tr>
    <tr><td colspan="3"><b>2018</b></td></tr>
    <tr><td> 23.5.2018   </td><td> Esteban&nbsp;Jenkins</td><td> Long read mapping algorithms</td></tr>
    <tr><td>             </td><td> Isidora&nbsp;Conic</td><td> Bioinformatics and Cancer</td></tr>
    <tr><td> 9.5.2018   </td><td> Marc&nbsp;Haber</td><td> History of the Near East from Ancient and Present-day Genome Sequences</td></tr>
    <tr><td> 11.4.2018   </td><td> Viorica&nbsp;Chelban</td><td> Disease discovery in neurogenetics: from patient to pathways and therapies</td></tr>
    <tr><td> 28.3.2018   </td><td> Marian&nbsp;Novotný</td><td> Fosforylace proteinů - bioinformatický pohled</td></tr>
    <tr><td> 7.3.2018   </td><td> Jan&nbsp;Pačes</td><td> Artificial life</td></tr>
    <tr><td> 3.1.2018   </td><td> David&nbsp;Nohejl</td><td> Variant Effect Prediction</td></tr>
    <tr><td colspan="3"><b>2017</b></td></tr>
    <tr><td> 20.12.2017   </td><td> Tomáš&nbsp;Gavenčiak</td><td> Genotype refinement  with PBWT</td></tr>
    <tr><td> 6.12.2017   </td><td> Audrey&nbsp;Lin</td><td> Rates of evolutionary change in ancient domesticated animals</td></tr>
    <tr><td> 22.11.2017   </td><td> Andrew&nbsp;Page</td><td>High throughput pathogen informatics</td></tr>
    <tr><td> 1.11.2017   </td><td> Martin&nbsp;Modrák</td><td>Genexpi: Discovering Regulatory Interactions on the GPU</td></tr>
    <tr><td> 18.10.2017   </td><td> David&nbsp;Dufour&nbsp;Rausell</td><td>A brief overview of Structural Bioinformatics: from proteins to chromosomes</td></tr>
    <tr><td> 3.5.2017   </td><td> Karel&nbsp;Sedlář</td><td> Vizualizace v metagenomice</td></tr>
    <tr><td> 19.4.2017   </td><td> Tereza&nbsp;Tolarová</td><td> Aktivace embryonálního genomu v preimplantačním vývoji savců</td></tr>
    <tr><td> 5.4.2017   </td><td> Daniel&nbsp;Morais</td><td> You can change the knowledge about the microbial universe </td></tr>
    <tr><td> 22.3.2017   </td><td> Matěj&nbsp;Lexa</td><td> pqsfinder: flexible detection of sequences forming G-quadruplexes in R/Bioconductor</td></tr>
    <tr><td> 8.3.2017   </td><td> Lukáš&nbsp;Hrbek</td><td> Variant calling with deep neural networks</td></tr>
    <tr><td> 11.1.2017   </td><td> Jiří&nbsp;Vohradský</td><td> Modeling and simulation of lambda-phage genetic network</td></tr>
    <tr><td> 4.1       </td><td> Ashkan Zareie<br>Victor Santiago</td><td> Drug-Target Interaction Prediction <br> Variant Calling  </td></tr>
    <tr><td colspan="3"><b>2016</b></td></tr>
    <tr><td> 14.12       </td><td> Jiří Macas</td><td> Repetitivní DNA: "Temná hmota" genomu a její výzkum pomocí teorie grafů</td></tr>
    <tr><td> 30.11       </td><td> Milan Malinský</td><td> Bioinformatické metody v evoluční genomice</td></tr>
    <tr><td> 16.11       </td><td> Michal Kolář</td><td> Evoluce zachycená v (proteinové) síti</td></tr>
    <tr><td> 2.11       </td><td> Holly Z. Bradley</td><td>The 1000 Genomes Project, what is it and can it help my research?</td></tr>
    <tr><td> 19.10      </td><td> Jan Kalina</td><td> Statistické klasifikační metody při celogenomové analýze genových expresí</td></tr>
    <tr><td> 5.10       </td><td> Jan Brezovský     </td> <td> Identifikace aktivních malých molekul pro biotechnologie</td></tr>
    <tr><td> 18.5       </td><td> Matej Pribis      </td><td> Copy Number Variation</td></tr>
    <tr><td> 11.5       </td><td> Erik Garrison     </td><td> Genome variation graphs and genotyping by tera-scale learning</td></tr>
    <tr><td> 4.5        </td><td> Karel Jalovec     </td><td> Discriminatory analysis of sequenced read-sets</td></tr>
    <tr><td> 27.4       </td><td> Radka Reifová     </td><td> Populační a speciační genomika: hledání genetické podstaty vzniku druhů na modelu dvou druhů slavíků.</td></tr>
    <tr><td> 20.4       </td><td> Stephan Schiffels </td><td> How genomes help us understand human history</td></tr>
    <tr><td> 13.4       </td><td> Petr Daněček      </td><td> Základní úloha bioinformatiky: Alignment and Mapping</td></tr>
    <tr><td> 5.4        </td><td> Jan Pačes         </td><td> Genom šimpanze, genom člověka</td></tr>
    <tr><td> 16.3       </td><td> David Hoksza        </td><td> Procházky chemickým prostorem</td></tr>
    <tr><td> 9.3        </td><td> Martin Loebl        </td><td> DNA repeats and information concealing</td></tr>
    <tr><td> 2.3        </td><td> Daniel Svozil       </td><td> Je DNA struktura nudná?</td></tr>
    <tr><td colspan="3"><b>2015</b></td></tr>
    <tr><td> 16.12      </td><td> Petr Maršálek       </td><td> Vědomí a jeho poruchy, pokus o objektivní popis</td></tr>
    <tr><td> 9.12       </td><td> P. Pančoška, J. Nešetřil <td> Matematika struktur pro informatiku personalizované medicíny</td></tr>
    <tr><td> 2.12       </td><td> Petr Daněček        </td><td> Copy Number Variation and HMM</td></tr>
    <tr><td> 25.11      </td><td> Viktor Stránecký    </td><td> Next Generation Sequencing</td></tr>
    <tr><td> 18.11      </td><td> Karel Fišer         </td><td> Leukémie s přesmykem</td></tr>
    <tr><td> 11.11      </td><td> Tomáš Sieger        </td><td> Počítejte (s) neurony!</td></tr>
    <tr><td> 4.11       </td><td> David Hoksza        </td><td> Identifikace strukturní podobnosti RNA molekul</td></tr>
    <tr><td> 19.10      </td><td> Jan Holub           </td><td> Burrows-Wheelerova transformace v bioinformatice</td></tr>
    <tr><td> 12.10      </td><td> Jan Pačes           </td><td> Analýza Endogenních Retrovirů</td></tr>
    </table>

</div>
</div>
</div>
</body>
</html>

